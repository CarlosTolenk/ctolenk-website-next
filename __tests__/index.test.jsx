import {render, screen} from '@testing-library/react'
import Home from '../pages/index'

describe('Home', () => {

    it('should renders homepage unchanged', () => {
        const {container} = render(<Home/>)
        expect(container).toMatchSnapshot()
    });


    // it('should renders a heading', () => {
    //     render(<Home description={'Description'} title={'Carlos'}/>)
    //
    //     const heading = screen.getByText('.site-title', {
    //         name: /Carlos Tolentino/i,
    //     })
    //
    //     expect(heading).toBeInTheDocument()
    // });
})
