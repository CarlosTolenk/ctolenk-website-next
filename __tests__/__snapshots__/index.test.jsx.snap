// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should renders homepage unchanged 1`] = `
<div>
  <header
    class="header"
  >
    <div
      class="header-content"
    >
      <div
        class="profile-picture-block"
      >
        <div
          class="my-photo"
        >
          <span
            style="box-sizing: border-box; display: block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px;"
          >
            <img
              alt="image"
              class="img-fluid"
              data-nimg="fill"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
            />
            <noscript />
          </span>
        </div>
      </div>
      <div
        class="site-title-block"
      >
        <div
          class="site-title"
        >
          Carlos Tolentino
        </div>
        <div
          class="type-wrap"
        >
          <br />
          <div
            class="typed-strings"
          >
            <span>
              Mobile Developer
            </span>
            <span>
              Fullstack Web Developer
            </span>
          </div>
          <span
            class="typed"
          />
        </div>
      </div>
      <div
        class="site-nav"
      >
        <ul
          class="header-main-menu"
          id="header-main-menu"
        >
          <li>
            <a
              class="active"
              href="#home"
            >
              <i
                class="fas fa-home"
              />
               Home
            </a>
          </li>
          <li>
            <a
              href="#about-me"
            >
              <i
                class="fas fa-user-tie"
              />
               About Me
            </a>
          </li>
          <li>
            <a
              href="#resume"
            >
              <i
                class="fas fa-award"
              />
               Resume
            </a>
          </li>
          <li>
            <a
              href="#portfolio"
            >
              <i
                class="fas fa-business-time"
              />
               Portfolio
            </a>
          </li>
          <li>
            <a
              href="#blog"
            >
              <i
                class="fas fa-book-reader"
              />
               Blog
            </a>
          </li>
          <li>
            <a
              href="#contact"
            >
              <i
                class="fas fa-paper-plane"
              />
               Contact
            </a>
          </li>
          <li>
            <a
              href="#appointments"
            >
              <i
                class="fas fa-calendar-check"
              />
               Appointments
            </a>
          </li>
        </ul>
        <div
          class="copyrights"
        >
          Â©
          2022
           All rights reserved.
        </div>
      </div>
    </div>
  </header>
</div>
`;
